app/views/trainees/contact_details/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<h2 class="govuk-heading-l">Contact details</h2>
+
+<%= form_with model: @trainee, data: { remote: false } do |f| %>
```

Comment:
This isn't rendering properly. `<form data-remote="false" action="/trainees/1" accept-charset="UTF-8" data-remote="true" method="post">` I think we can remove the `data: {remote: false}`. That gets added by default.

Reply:
nice spot. dev tools lies but the source tells the truth. I've added a possibly controversial commit that fixes the issue